server/router/src/test/java/org/infinispan/server/router/utils/HotRodClientTestingUtil.java

```diff
@@ -0,0 +1,60 @@
+package org.infinispan.server.router.utils;
+
+import java.net.InetAddress;
+
+import org.infinispan.client.hotrod.RemoteCacheManager;
+import org.infinispan.client.hotrod.configuration.ConfigurationBuilder;
+
+public enum HotRodClientTestingUtil {
+    INSTANCE;
```

Comment:
I wanted to use those like singletons and Josh says this is the best way to do it: http://www.drdobbs.com/jvm/creating-and-destroying-java-objects-par/208403883?pgno=3.

However as I can see, nobody likes this approach so let me refactor this to static.